projects/step-frontend/src/lib/modules/timeseries/chart/model/ts-chart-settings.ts

```diff
@@ -17,5 +18,6 @@ export interface TSChartSettings {
 export interface TSChartSeries extends Series {
   id: string;
   data: (number | null | undefined)[];
+  metadata?: any[];
```

Comment:
According to the code, metadata contains an array of bucket's attributes.
If so, it's type can be defined as
```suggestion
  metadata?: (BucketResponse['attributes'])[];
```

Or for easier usage in other places, you can define an alias
`type MetadataItem = BucketReponse['attributes']`